executor/internal/vecgroupchecker/vecgroupchecker.go

```diff
@@ -0,0 +1,514 @@
+// Copyright 2023 PingCAP, Inc.
```

Comment:
I think `vec_group_checker.go` is better.

Reply:
updated